miro_preprocessor/terraform.tf

```diff
@@ -18,3 +18,13 @@ data "terraform_remote_state" "platform" {
     region = "eu-west-1"
   }
 }
+
+data "terraform_remote_state" "lambda" {
```

Comment:
This is only here beacuse of the `lambda_error_alarm_arn` output. I wonder whether this should be in the shared infra instead

Reply:
👍  I think we can consolidate this after.